### DISCUSSION TITLE
Is there a reason the RendererPlugin interface isn't exported as a part of the typescript types?

### DISCUSSION THREAD

**Discussion by mccolljr** [excaliburjs/Excalibur#3233] [Q&A] [answered]
I wanted to play around with a custom renderer plugin, but I'm noticing that the RendererPlugin type isn't exported. TypeScript complains that the type is unexported, and some digging indicates that it is exported from `renderer.d.ts` but not from the `Context` `index.d.ts`. Is this intentional or just an oversight?

**Comment by eonarheim** [comment, marked as answer]
Howdy @mccolljr! 

Feels like an oversight to me, I'll convert this to an issue,. Having those types public would be helpful for anyone implementing a custom renderer

**Reply by eonarheim** [reply]
https://github.com/excaliburjs/Excalibur/issues/3234
